packages/thirdweb/src/indexer/endpoints/getReceipts.ts

```diff
@@ -0,0 +1,6 @@
+// TODO clarify the types, no need to sanitize here because we are already doing it server-side?
+export async function getReceipts(client: any, params: any) {
```

Comment:
Do we want to call this `getTransactions` instead?

Reply:
Changed in [implement getTransactions for chainsaw](https://github.com/thirdweb-dev/js/pull/3741/commits/c01748cacd8b9de2aadeabb9bd5b86d4b6815984)